configure

```diff
@@ -731,11 +731,14 @@ def configure_library(lib, output):
     if pkg_cflags:
       output['include_dirs'] += (
           filter(None, map(str.strip, pkg_cflags.split('-I'))))
+    elif options.__dict__[shared_lib + '_includes']:
```

Comment:
@bnoordhuis Oh, you are right. In that case, we can write this as `if getattr(options, shared_lib + '_includes'):`. But this also will work. So, I think its okay.